src/components/LoginDialog.jsx

```diff
@@ -1,33 +1,48 @@
 import React from 'react';
-import Button from '@material-ui/core/Button';
 import Dialog from '@material-ui/core/Dialog';
 import DialogContent from '@material-ui/core/DialogContent';
 import DialogContentText from '@material-ui/core/DialogContentText';
 import DialogTitle from '@material-ui/core/DialogTitle';
 
-import FirebaseLoginScreen from './FirebaseLoginScreen';
+import firebase from 'firebase';
+import StyleFirebaseAuth from 'react-firebaseui/StyledFirebaseAuth';
+import PropTypes from 'prop-types';
 
-export default function LoginDialog() {
-  const [open, setOpen] = React.useState(false);
-  const handleClickOpen = () => setOpen(true);
-  const handleClose = () => setOpen(false);
+class LoginDialog extends React.Component {
```

Comment:
This can be written as a [function component](https://reactjs.org/docs/components-and-props.html)

Reply:
lines 11-13 make them as a function?

Reply:
Oh wait you mean `LoginDialog` to be written as a function rather than a class?

Reply:
And you would place lines 11 - 13 inside of your LoginDialog function